apps/fz_http/lib/fz_http_web/channels/gateway_channel.ex

```diff
@@ -0,0 +1,38 @@
+defmodule FzHttpWeb.Gateway.Channel do
```

Comment:
nit: naming convention to match `NotificationChannel`

```suggestion
defmodule FzHttpWeb.GatewayChannel do
```